README.md

```
@@ -182,21 +182,40 @@ uvicorn main:app --reload
+1. **Open Supabase Dashboard**
+   - Go to Supabase Dashboard (https://supabase.com/) and log in.
    - Select your created project.
 
 2. **Access the SQL Editor**
 
    - In the left sidebar, click on **SQL Editor**.
 
-3. **Run the SQL Script**
+3. **Create Required Tables**
+   - Run the following SQL in the Supabase SQL Editor:
+```sql
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)